src/api/app/models/event/base.rb

```diff
@@ -263,6 +263,18 @@ def _roles(role, project, package = nil)
       p ||= ::Project.find_by_name(project)
       obj_roles(p, role)
     end
+
+    def create_project_log_entry
```

Comment:
@mdeniz do you mind if we do this in a PR after this? Since its harder than it looks to write an isolated unit test for this method and this is a fix for a live bug.